auth/test.py

```diff
@@ -0,0 +1,48 @@
+from unittest import TestCase
+
+import flask_login
+
+from blog.views import *
+from .views import *
+from user.views import *
+
+
+class AuthTest(TestCase):
+    def setUp(self):
+        self.client = app.test_client()
+        self.user = User(email='test@email.com').set_password('test').save()
+
+    def tearDown(self):
+        User.objects.delete()
```

Comment:
We should ensure we are running a test database with this command.
Not sure if we should go for documentation, default settings, or other solution